frontend/amundsen_application/static/js/ducks/search/filters/reducer.ts

```diff
@@ -46,10 +47,39 @@ export interface ResourceFilterReducerState {
   };
 }
 
+export function getDefaultFiltersForResource(
+  resourceType: ResourceType
+): ResourceFilterReducerState {
+  const filterCategories = getFilterConfigByResource(resourceType);
+  const initialValue = {};
+  const defaultFilters =
+    filterCategories
+      ?.filter(
+        (filterCategory) =>
+          filterCategory.defaultValue && filterCategory.defaultValue !== []
```

Comment:
```suggestion
        ({defaultValue) =>
          defaultValue && defaultValue.length
```

We don't usually compare to empty arrays in JavaScript, because:
```
[] === []
false
``` 
:)